internal/pkg/utils/reflect.go

```diff
@@ -46,6 +46,19 @@ func MapFromOneTaggedField(tag string, model interface{}) *orderedmap.OrderedMap
 	return reflection.FieldByName(field.Name).Interface().(*orderedmap.OrderedMap)
 }
 
+func StringFromOneTaggedField(tag string, model interface{}) (str string, found bool) {
```

Comment:
Ak sa zada `tag=DescriptionFileTag` ... tak to vrati `description` z `model` ako string.